File: OOP/tic-tac-toe-board-game-app/src/com/techlab/game/test/GameTest.java
package com.techlab.game.test;

import java.util.Scanner;

import com.techlab.Exception.AllReadyMarkCellException;
import com.techlab.board.Mark;
import com.techlab.game.Game;
import com.techlab.player.Player;

public class GameTest {

	public static void main(String args[]) throws AllReadyMarkCellException {
		Player player1 = new Player("Darshan", Mark.O);
		Player player2 = new Player("Vivek", Mark.X);

		Scanner inputPosition = new Scanner(System.in);
		
		int cellPosition;

		Game game = new Game(player1, player2);
		game.startGame();
				
		while (game.getResult().equals("In_Progress")) {
			System.out.println("entered position : ");
			cellPosition = inputPosition.nextInt();
			game.setMarkInCellOfBoard(cellPosition);
		}
		System.out.println(game.getResult());
	}
	
}

File: nodeJs/tic-tac-toe-board-game/gameTest.js
let Player = require('./player');
let Mark = require('./mark');
let Game = require('./game');
let readlineSync = require('readline-sync');

let cellPosition;

let player1 = new Player();
let player2 = new Player();

let game = new Game(player1, player2);

let firstPlayerName = readlineSync.question('Enter First Player Name : ');
player1.name = firstPlayerName;
player1.mark = Mark.O;
let secondPlayerName = readlineSync.question('Enter Second Player Name : ');
player2.name = secondPlayerName;
player2.mark = Mark.X;

game.startGame();

while (game.getResult() == "In_Progress") {
    cellPosition = readlineSync.question('entered position : ');
    game.setMarkInCellOfBoard(cellPosition);

    game.displayBoard();
}
console.log(game.getResult());